mne/decoding/tests/test_base.py

```diff
@@ -20,7 +63,9 @@ def test_get_coef():
     from sklearn.base import TransformerMixin, BaseEstimator
     from sklearn.pipeline import make_pipeline
     from sklearn.preprocessing import StandardScaler
-    from sklearn.linear_model import LinearRegression
+    from sklearn.linear_model import Ridge
+
+    lm = LinearModel(Ridge())
```

Comment:
When using Ridge regression with `alpha=1`, you don't reconstruct the patterns exactly anymore. Why use Ridge instead of OLS regression?